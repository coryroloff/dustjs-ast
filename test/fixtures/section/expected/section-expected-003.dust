{#standard}{#inner-standard}{/inner-standard}{/standard}