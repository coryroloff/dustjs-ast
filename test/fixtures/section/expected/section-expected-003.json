{
	"type": "Template",
	"body": [
		{
			"type": "Section",
			"body": [],
			"params": [],
			"id": {
				"type": "Identifier",
				"path": [
					{
						"type": "Key",
						"name": "exists",
						"loc": {
							"start": {
								"line": 1,
								"column": 3
							},
							"end": {
								"line": 1,
								"column": 9
							}
						}
					}
				],
				"loc": {
					"start": {
						"line": 1,
						"column": 3
					},
					"end": {
						"line": 1,
						"column": 9
					}
				}
			},
			"kind": "exists",
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 19
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Punctuator",
			"value": "{",
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 2
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "?",
			"loc": {
				"start": {
					"line": 1,
					"column": 2
				},
				"end": {
					"line": 1,
					"column": 3
				}
			}
		},
		{
			"type": "Key",
			"value": "exists",
			"loc": {
				"start": {
					"line": 1,
					"column": 3
				},
				"end": {
					"line": 1,
					"column": 9
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"loc": {
				"start": {
					"line": 1,
					"column": 9
				},
				"end": {
					"line": 1,
					"column": 10
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "{",
			"loc": {
				"start": {
					"line": 1,
					"column": 10
				},
				"end": {
					"line": 1,
					"column": 11
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "/",
			"loc": {
				"start": {
					"line": 1,
					"column": 11
				},
				"end": {
					"line": 1,
					"column": 12
				}
			}
		},
		{
			"type": "Key",
			"value": "exists",
			"loc": {
				"start": {
					"line": 1,
					"column": 12
				},
				"end": {
					"line": 1,
					"column": 18
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"loc": {
				"start": {
					"line": 1,
					"column": 18
				},
				"end": {
					"line": 1,
					"column": 19
				}
			}
		}
	],
	"loc": {
		"start": {
			"line": 1,
			"column": 1
		},
		"end": {
			"line": 1,
			"column": 19
		}
	}
}